main.go

```diff
@@ -13,20 +14,28 @@ import (
 )
 
 var (
-	app                 = kingpin.New("svu", "semantic version util")
-	nextCmd             = app.Command("next", "prints the next version based on the git log").Alias("n").Default()
-	majorCmd            = app.Command("major", "new major version")
-	minorCmd            = app.Command("minor", "new minor version").Alias("m")
-	patchCmd            = app.Command("patch", "new patch version").Alias("p")
-	currentCmd          = app.Command("current", "prints current version").Alias("c")
-	pattern             = app.Flag("pattern", "limits calculations to be based on tags matching the given pattern").String()
-	prefix              = app.Flag("prefix", "set a custom prefix").Default("v").String()
-	stripPrefix         = app.Flag("strip-prefix", "strips the prefix from the tag").Default("false").Bool()
-	preRelease          = app.Flag("pre-release", "adds a pre-release suffix to the version, without the semver mandatory dash prefix").String()
-	build               = app.Flag("build", "adds a build suffix to the version, without the semver mandatory plug prefix").String()
-	directory           = app.Flag("directory", "specifies directory to filter commit messages by").Default("").String()
-	tagMode             = app.Flag("tag-mode", "determines if latest tag of the current or all branches will be used").Default("current-branch").Enum("current-branch", "all-branches")
-	forcePatchIncrement = nextCmd.Flag("force-patch-increment", "forces a patch version increment regardless of the commit message content").Default("false").Bool()
+	app           = kingpin.New("svu", "semantic version util")
+	nextCmd       = app.Command("next", "prints the next version based on the git log").Alias("n").Default()
+	majorCmd      = app.Command("major", "new major version")
+	minorCmd      = app.Command("minor", "new minor version").Alias("m")
+	patchCmd      = app.Command("patch", "new patch version").Alias("p")
+	currentCmd    = app.Command("current", "prints current version").Alias("c")
+	preReleaseCmd = app.Command("prerelease", "new pre release version based on the next version calculated from git log").
+			Alias("pr")
+	pattern     = app.Flag("pattern", "limits calculations to be based on tags matching the given pattern").String()
+	prefix      = app.Flag("prefix", "set a custom prefix").Default("v").String()
+	stripPrefix = app.Flag("strip-prefix", "strips the prefix from the tag").Default("false").Bool()
+	preRelease  = app.Flag("pre-release", "adds a pre-release suffix to the version, without the semver mandatory dash prefix").
+			String()
+	build = app.Flag("build", "adds a build suffix to the version, without the semver mandatory plug prefix").
+		String()
```

Comment:
these could probably be added to `preReleaseCmd` only

Reply:
I was trying to ensure this was a non breaking change for others already using this in their workflows.

I agree that adding the --pre-release flag to preReleaseCmd makes sense, if the build flag is added to that too then it would prevent someone from using svu to create a version such as `v1.0.0+33` so I'll defer to you if you're ok with removing that ability.

Reply:
zah, right, good point... lets do it only with the new flag then!